BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -149,6 +150,7 @@ public void Start()
             }
             targetsAssigned = new List<TargetInfo>();
             StartCoroutine(DelayedStart());
+            if (isClusterMissile && vessel.Parts.Count == 1) isLaunchedClusterMissile = true;
```

Comment:
This also needs a `HighLogic.LoadedSceneIsFlight` condition as `vessel` isn't valid in the SPH.